runtimes/pythonrt/requirements.txt

```diff
@@ -12,6 +12,10 @@ redis ~= 5.0
 slack-sdk ~= 3.27
 twilio ~= 9.0
 
+# AutoKitteh SDK
+
+autokitteh ~= 0.1
```

Comment:
Given that we own this, doesn't it make sense to always get the latest version instead of pinning?

Reply:
I want to give users the option when to upgrade. If we make breaking change to `autokitteh` and they still want to run on older version - it's a problem.